pkg/util/escape.go

```diff
@@ -20,6 +20,19 @@ import (
 	"strings"
 )
 
+// EscapeQualifiedPluginName converts a plugin name in the format
+// vendor/pluginname into a proper ondisk vendor~pluginname plugin directory
+// format.
+func EscapePluginName(in string) string {
```

Comment:
At the moment `EscapePluginName(...)` is identical to `EscapeQualifiedNameForDisk(...)` and `UnescapePluginName(...)` is identical to `UnescapeQualifiedNameForDisk(...)`. Instead of duplicating the functionality, just rename the original methods to something more generic, and use them. If the actual functionality ever changes, then we can consider breaking them out into separate methods.